### PR TITLE
Fixes issue #183

### PR DESCRIPTION
**Pull request by RickSBrown**
Actually this is a workaround, not a fix because the bug is in closure compiler. I'll break it down in detail below.

First understand that the function `axs.properties.findTextAlternatives` can legitimately return `null`.
Then take a look at the way we call it:

```
var textAlternatives = axs.properties.findTextAlternatives(element, {});
if (!textAlternatives || textAlternatives.trim() === '')
```

All good. But then take a look at what closure compiler produces:
```
return "" === axs.properties.findTextAlternatives(a, {}).trim() ? !1 : !0;
```
That will throw an exception if the return value is null.
The easiest workaround is to get closure compiler to produce something else. The proposed change does not change the logic of the audit but results in safe code generated by closure compiler:
```
a = axs.properties.findTextAlternatives(a, {});
  return null == a || "" === a.trim() ? !1 : !0;
```

### CODE REVIEW

**Comment by RickSBrown** [comment]
@alice PTAL

File: src/audits/FocusableElementNotVisibleAndNotAriaHidden.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ axs.AuditRules.addRule({
         // Ignore elements which have a negative tabindex and no text content,
         // as they will be skipped by assistive technology
         var textAlternatives = axs.properties.findTextAlternatives(element, {});
-        if (!textAlternatives || textAlternatives.trim() === '')
+        if (textAlternatives == null || textAlternatives.trim() === '')
```
Comment (alice) [inline src/audits/FocusableElementNotVisibleAndNotAriaHidden.js]: Let's use === null - closure compile should catch if `findTextAlternatives` weirdly returns undefined anywhere.

**Comment by alice** [comment]
Please update the Changelog: https://github.com/GoogleChrome/accessibility-developer-tools/blob/master/Changelog.md - manual process to help me write release notes. Sorry for the overhead :(

Once that's done and the nit addressed, this is :+1: - definitely an improvement regardless of the closure weirdness.

**Comment by RickSBrown** [comment]
No worries about the changelog overhead, it is well worth the effort IMHO.

**Pull request merged**